src/main/java/frc/robot/Constants.java

```diff
@@ -69,5 +69,18 @@ public static final class Swerve {
     public static final Translation2d BR_LOCATION = new Translation2d(-APOTHEM, -APOTHEM);
     public static final Translation2d BL_LOCATION = new Translation2d(-APOTHEM, APOTHEM);
   }
-
+  public static final class Climb{
+    public static final double MAX_UP_VOLTAGE = 0.4; 
+    public static final double MAX_DOWN_VOLTAGE= -0.4; 
+    public static final double EXTEND_MOTOR_SETPIONT = 0.9; 
+    public static final double RETRACT_MOTOR_SETPOINT = 0.9;
+    public static final int RCLIMB_MOTOR_ID = 14;
+    public static final int LCLIMB_MOTOR_ID = 15;
+    public static final double CLIMB_kP = .1;
+    public static final double CLIMB_kI = .1;
+    public static final double CLIMB_kD = .1;
```

Comment:
The P, I, and D gains should be left as zero for now.

Reply:
solved